### ISSUE TITLE
Please add `_WebElementWrapper.wait_for_element()` method to support  relative selector

### ISSUE THREAD

**Issue by vimagick** [horejsek/python-webdriverwrapper#12]
```python
>>> parent = driver.get_elm(id_='parent')
>>> child = parent.wait_for_element(xpath='./relative/xpath/to/child')
'_WebElementWrapper' object has no attribute 'wait_for_element'
```

**Comment by horejsek** [comment]
Ok, I will do it with your #11 this week.

**Comment by vimagick** [comment]
You're awesome and cool! 😎 👍

**Comment by horejsek** [comment]
Fixed in version 2.5.0.
